src/docs/sphinx/developerGuide/Contributing/InstallWin.rst

```diff
@@ -0,0 +1,266 @@
+.. _InstallWin:
+
+Installing GEOSX on Windows machines using Docker
+===================================================
+
+In this section, we will install GEOSX on a Windows machine using a _docker_ container with a precompiled version of
+GEOSX's third party libraries (TPL). These steps are an adaptation of `ref:UsingDocker` for the Windows environment.
+
+The different versions of the container can be found on `docker hub <https://hub.docker.com/r/geosx/ubuntu18.04-gcc8/tags?page=1&ordering=last_updated>`_.
+Note that to work on the latest version of the code, you'll need to use the most recently pushed container.
+
+
+1. Install Docker Desktop
+-------------------------
+
+On your Windows machine, follow these `steps <https://docs.docker.com/desktop/windows/install/>`_.
+Download the most recent installer for Docker Desktop. Before installation please check the current status of Windows
+Subsystem Linux (WSL) on your machine as Docker will use *WSL2* as a backend. To do that, open a *PowerShell(Admin)*
+
```

Comment:
@TotoGaz I think there is not a strict need for *WSL2* but strongly recommended. See [here](https://docs.docker.com/desktop/windows/wsl/)

Reply:
Great! I think it should be nice to mention this page then.
In a few years, the doc maintainer will know why we asked for WSL2! 🤣

Reply:
Hopefully by the next update there will be even more linux in windows that this will be history. :smile: